var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -246,6 +253,30 @@ def check_compilers(self):
                 + " has no Fortran compiler added in spack. Add it or use openblas~fortran!"
             )
 
+    @property
+    def headers(self):
+        # As in netlib-lapack, the only public headers for cblas and lapacke in
+        # openblas are cblas.h and lapacke.h. The remaining headers are private
+        # headers either included in one of these two headers, or included in
+        # one of the source files implementing functions declared in these
+        # headers.
+        return find_headers(["cblas", "lapacke"], self.prefix.include)
+
+    @property
+    def libs(self):
+        spec = self.spec
+
+        # Look for openblas{symbol_suffix}
+        name = "libopenblas, openblas"
```

Comment:
Note that the first argument of `find_libraries` allows a string, but only if that string represents a single library. If you want to look for multiple, you need to break it into a list of strings.

Also, you'd need to individually append the symbol suffix anyway. That being said, I don't see any `symbol_suffix` variant in `boost` right now and I don't see this diff adding one. Without the `symbol_suffix` logic, this matches the behavior of `Spec._libs_default_handler`.